src/wp-includes/theme-templates.php

```diff
@@ -118,14 +118,8 @@ function the_block_template_skip_link() {
 	if ( ! $_wp_current_template_content ) {
 		return;
 	}
-	?>
 
-	<?php
-	/**
-	 * Print the skip-link styles.
-	 */
-	?>
-	<style id="skip-link-styles">
+	$skip_link_styles = '
```

Comment:
Can take advantage of PhpStorm language injection:

```suggestion
	$skip_link_styles = /* language=CSS */ '
```

Before | After
-------|------
![image](https://github.com/WordPress/wordpress-develop/assets/134745/306f65dc-90f2-4738-b184-16de6967e037) | ![image](https://github.com/WordPress/wordpress-develop/assets/134745/b5a6de78-55d3-4e10-abcc-8e8f0685336e)